jwst/tweakreg/tweakreg_catalog.py

```diff
@@ -71,7 +71,11 @@ def make_tweakreg_catalog(model, kernel_fwhm, snr_threshold, sharplo=0.2,
                             sharplo=sharplo, sharphi=sharphi, roundlo=roundlo,
                             roundhi=roundhi, brightest=brightest,
                             peakmax=peakmax)
-    sources = daofind(model.data)
+
+    # Mask the non-imaging area (e.g. MIRI)
+    mask = (model.dqflags.pixel['NON_SCIENCE'] & model.dq).astype(np.bool)
```

Comment:
This is a bug.  The model itself does not have `dqflags`.  One can get the dict from `datamodels` though.
```
from jwst import datamodels

mask = (datamodels.dqflags.pixel['NON_SCIENCE'] & model.dq).astype(np.bool)
```

Reply:
Fixed, thanks!